lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -143,6 +143,33 @@ def self.get_network_mbps(throughput)
       network_mbps
     end
 
+    # Take in string from amazon pricing api, return network properties
+    def self.get_network_information(network_string)
+      throughput = nil
+      case network_string
+        when 'Very Low'
+          throughput = :very_low
+        when 'Low'
+          throughput = :low
+        when 'Low to Moderate'
+          throughput = :low_to_moderate
+        when 'Moderate'
+          throughput = :moderate
+        when 'High'
+          throughput = :high
+        when '10 Gigabit','Up to 10 Gigabit'
+          throughput = :ten_gigabit
+        when '20 Gigabit'
+          throughput = :twenty_gigabit
+        when '25 Gigabit'
+          throughput = :twentyfive_gigabit
+        else
+          binding.pry
```

Comment:
debug.
is the return expected to be a 'symbol' or a 'string'?

Reply:
We convert it to a string below. The network MBPS gives us a int value.